evennia/commands/default/muxcommand.py

```diff
@@ -109,15 +114,44 @@ def parse(self):
             else:
                 args = ""
                 switches = switches[0].split('/')
+            # Parse mux options, comparing them against user-provided switches, expanding abbreviations.
```

Comment:
I originally had done that, but I took the example of `account_caller`, which is also optional, but not set to any default.

I agree, it is cleaner to default these values instead of testing to see if they were assigned.

Reply:
`account_caller`, as I recall, is used by a special group of Commands, those defined in the `AccountCmdSet`. At the time of its implementation the whole concept of Character/Account was new. One could certainly argue that it could be useful existing as a default too.

Reply:
I addressed this with `account_caller`, too, and I think it turned out well.